v2/cn/download.md.tt2

```diff
@@ -1,146 +1,182 @@
 <!---
     @title         下载
-    @creator       Yichun Zhang
+    @creator       章亦春
+    @Translator    何伟平
     @created       2011-06-21 04:25 GMT
 --->
 
-对于一些常见的 Linux 发行版本，OpenResty<sup>&reg;</sup> 提供 [官方预编译包](linux-packages.html)。确保你首先用这种方式来安装。
+# 二进制版本
 
-你可以下载下面这些源码包。然后按照 [安装](installation.html)页面的说明来 build 和安装。
+## Linux
 
-所有的版本都已经使用 Yichun Zhang 的 PGP 公钥 [A0E98066](https://pgp.mit.edu/pks/lookup?op=get&search=0xB550E09EA0E98066) 签名。
+OpenResty<sup>&reg;</sup> 给常见 Linux 发布提供[官方预编译安装包](linux-packages.html)。 
+
+目前我们支持 Ubuntu, Debian, RHEL, CentOS, Fedora, 和 Amazon Linux。
+
+我们还给用户提供官方的包仓库，可以很容易检索和安装、更新包。
+
+## Win32
+
+所有版本都是用章亦春的 PGP 公钥
```

Comment:
Better keep the original name `Yichun Zhang` since it is the English name that is included in the GPG key. Better not translate that name :)